.github/actions/spelling/allow/allow.txt

```diff
@@ -75,26 +64,18 @@ rlig
 rubyw
 runtimes
 servicebus
-shcha
-similaritytolerance
 slnt
 stakeholders
 subpage
 sustainability
 sxn
-TLDR
```

Comment:
This is removed by correcting the one instance to `TL;DR` (see src/types/UiaTextRangeBase.cpp)